Tasks/CargoAuthenticateV0/package.json

```diff
@@ -24,12 +24,12 @@
     },
     "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
     "dependencies": {
-        "@types/node": "^10.17.0",
+        "@types/node": "^16.11.39",
         "@types/mocha": "^5.2.7",
         "@types/uuid": "^8.3.0",
         "azure-pipelines-tasks-artifacts-common": "1.0.1",
-        "azure-pipelines-tasks-packaging-common": "3.206.0",
-        "azure-pipelines-task-lib": "^3.1.7",
+        "azure-pipelines-tasks-packaging-common": "3.214.0",
```

Comment:
need to add "node_modules/azure-pipelines-tasks-packaging-common/node_modules/azure-pipelines-task-lib" in make.json for deleting this library and have one version of azure-pipelines-task-lib

Reply:
Do you mean this?
https://github.com/microsoft/azure-pipelines-tasks/blob/945021505008b7c0a4ebf345fe923f8c233d32a0/Tasks/CargoAuthenticateV0/make.json#L5

Reply:
yup, you need to add the same for azure-pipelines-tasks-packaging-common, because it's using [azure-pipelines-task-lib](https://github.com/microsoft/azure-pipelines-tasks-common-packages/blob/main/common-npm-packages/packaging-common/package.json#L24) as well, in case if versions of the lib will be different somehow, for example, if in the future in packaging common version of azure-pipelines-task-lib will be 5 with a same major version of azure-pipelines-tasks-packaging-common and cargo still will have version 4, better to remove all nested azure-pipelines-task-lib except the version in package.json of the task

Reply:
Understood, added

Reply:
thanks!